e107_core/templates/bootstrap5/login_template.php

```diff
@@ -0,0 +1,71 @@
+<?php
+// $Id$
+
+if (!defined('e107_INIT')) { exit; }
+
+
+// Starter for v2. - Bootstrap 
+$LOGIN_TEMPLATE['page']['header'] = "
+	<div id='login-template'>
+		<div class='m-auto'>
+			{LOGO: login}
+		</div>";
+
+$LOGIN_TEMPLATE['page']['body'] = '
+		{LOGIN_TABLE_LOGINMESSAGE}
+        <h2 class="form-signin-heading">{LAN=LOGIN_4}</h2>';
+	if (e107::pref('core', 'password_CHAP') == 2)
+	{
+		$LOGIN_TEMPLATE['page']['body'] .= "
+    	<div style='text-align: center' id='nologinmenuchap'>"."Javascript must be enabled in your browser if you wish to log into this site"."
+		</div>
+    	<span style='display:none' id='loginmenuchap'>";
+	}
+	else
+	{
+	  $LOGIN_TEMPLATE['page']['body'] .= "<span>";
+	}
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_USERNAME'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_PASSWORD'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_SECIMG'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_TEXTBOC'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_REMEMBERME'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SUBMIT'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_FOOTER_USERREG'] = '<div class="row my-2 align-items-center">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_LOGINMESSAGE'] = "<div class='alert alert-danger'>{---}</div>";
+
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SIGNUP_LINK'] = '<div class="login-page-signup-link">{---}</div>';
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_FPW_LINK'] = '<div class="login-page-fpw-link">{---}</div>';
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_SECIMG']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4'>&nbsp;</label>
+	<div class='col-md-8'>{---}</div>
+</div>";
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_TEXTBOC']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4 col-form-label' for='code-verify'>" . e107::getSecureImg()->renderLabel() . "</label>
```

Comment:
```e107::getSecureImg()->renderLabel()``` needs to be placed into a shortcode

Reply:
Weird. I was sure I used core templates as the starting point and it looks like I didn't.

Reply:
@CaMer0n  This solution is used in core signup template too so I did it the same way. .

Reply:
@Jimmi08 Okay. It's good if we get both cleaned up to use shortcodes.